PENDING.md

```diff
@@ -38,6 +44,8 @@ FEATURES
 * Gaia CLI  (`gaiacli`)
   * [cli] Cmds to query staking pool and params
   * [gov][cli] #2062 added `--proposal` flag to `submit-proposal` that allows a JSON file containing a proposal to be passed in
+  * \#2040 Add `--bech` to `gaiacli keys show` and respective REST endpoint to
```

Comment:
We are still defaulting to bech32 so? Also, didn't we agreed, that sowing hex addresses is discouraged and all addresses are always shown in bech32 format?